src/workflows/workflows.ts

```diff
@@ -55,6 +60,15 @@ import {
   L2Signer,
 } from '../types';
 import { Registration__factory } from '../contracts';
+import { WalletConnection } from '../types/index';
+
+export type CreateOrderWithSignerRequest = WalletConnection & {
```

Comment:
Shouldn't we receive the wallet as a single attribute? Something like:
```
export type CreateOrderWithSignerRequest = {
  wallet: WalletConnection,
  request: GetSignableOrderRequest
}
```

Reply:
I don't incline to do that as it would be another layer of deconstruction. I think our discussion was based on we didn't wrap the whole params as an object. but if we do, I'd rather do this. 
Or, we could do as it is makes sense because it contains with only 2 params

```
createOrderWithSigner(wallet: WalletConnection, request: GetSignableOrderRequest)
```